PurchasesTests/Mocks/MockSubscriberAttributesManager.swift

```diff
@@ -5,6 +5,9 @@
 
 @testable import RevenueCat
 
+// swiftlint:disable identifier_name
+// swiftlint:disable large_tuple
```

Comment:
In case you were curious about why all of these are tuples and not structs, it's just us being lazy and [using a mock generator](https://github.com/seanhenry/MockGenerator), which generates these tuples

Reply:
yeah, I think it's a mock so it's not a real problem. WDYT?

Reply:
absolutely, it's never been an issue. Just wanted to provide context in case it seemed like a weird design choice

Reply:
Great! thanks for the context 🙂